backend/src/main/scala/com/softwaremill/adopttapir/starter/formatting/GeneratedFilesFormatter.scala

```diff
@@ -2,14 +2,18 @@ package com.softwaremill.adopttapir.starter.formatting
 
 import cats.effect.IO
 import com.softwaremill.adopttapir.starter.files.FilesManager
+import com.softwaremill.adopttapir.starter.files.StorageConfig
 import com.softwaremill.adopttapir.template.{CommonObjectTemplate, GeneratedFile}
 import com.typesafe.scalalogging.StrictLogging
 import org.scalafmt.interfaces.{Scalafmt, ScalafmtReporter}
 
 import java.io.{OutputStreamWriter, PrintWriter}
 import java.nio.file.Path
 
-class GeneratedFilesFormatter(fm: FilesManager) extends StrictLogging {
+final case class GeneratedFilesFormatter(storageConfig: StorageConfig) extends StrictLogging:
+
+  private val filesManager: FilesManager = FilesManager(storageConfig)
+  export filesManager.*
```

Comment:
why would a class whose responsibility is formatting, include methods for managing files? shouldn't these remain separate?

Reply:
That was a failed try to simplify things but yes defo formatter should have been separated from manager and then eventually some facade built on top of both. Anyway, I reversed back to previous solution.

Reply:
thanks :)